src/components/oParl/Person.tsx

```diff
@@ -54,6 +59,18 @@ export const Person = ({ data, navigation }: Props) => {
       <Row left={personTexts.formOfAddress} right={formOfAddress} leftWidth={leftWidth} />
       <Row left={personTexts.givenName} right={givenName} leftWidth={leftWidth} />
       <Row left={personTexts.familyName} right={familyName} leftWidth={leftWidth} />
+      <Row
+        left={personTexts.faction}
+        right={faction?.organization && getOrganizationNameString(faction.organization)}
+        leftWidth={leftWidth}
+        onPress={() =>
+          navigation.push('OParlDetail', {
+            id: faction?.id,
+            type: faction?.type,
+            title: texts.oparl.membership.membership
```

Comment:
`membership.membership`? could there be a different subkey for better readability? i have not checked the texts file to see myself, sorry for that.